youpin.js

```diff
@@ -43,7 +43,7 @@ module.exports = (m, api, conversation, apiUserId) => {
   function tagReplies(context){
     let tags = [ m.createQuickReplyButton( context.__('#done'), 'isEnding' ) ];
 
-    let categoryTags = _.map( categories, cat => {
+    let categoryTags = _.map( categories.filter( cat => context.hashtags.indexOf(cat) < 0 ), cat => {
```

Comment:
I think `context.hastags` is language dependent, and you will need this instead: 

`cat => context.hashtags.indexOf(context.__(cat)) < 0`

Also, you might need to reformat it to avoid lint error.